source/margarita/pillar.rst

```diff
@@ -0,0 +1,19 @@
+Salt Pillar
```

Comment:
It would also be nice to briefly describe the various files in the `conf/pillar` directory: project.sls, devs.sls, staging.sls, etc.